packages/commonwealth/client/scripts/state/api/communities/toggleCommunityStar.ts

```diff
@@ -23,20 +24,33 @@ const toggleCommunityStar = async ({ community }: ToggleCommunityStarProps) => {
 };
 
 const useToggleCommunityStarMutation = () => {
+  const user = useUserStore();
```

Comment:
it looks like the list of fav communities is messed up. Looks like index-related error. not sure if this is because of your PR though

https://github.com/hicommonwealth/commonwealth/assets/14819225/207e2e4a-8a50-4435-b7d4-bd73bda40786

Reply:
i remember fixing this in one of these commits

<img width="1223" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/a1d6753d-8a1e-40d8-bf8e-73039fe8d4c3">

can you try with lastest pull?

Reply:
it is still messed up for me

Reply:
fixed, please try again @masvelio

Reply:
Confirmed fixed.